### ISSUE TITLE
SpaceAroundMapEntryColon regex

### ISSUE THREAD

**Issue by PJHRobles** [nvuillam/npm-groovy-lint#51]
I've been trying to lint some of our files and one of the errors I'm getting is the `spaceAroundMapEntryColon` Rule. Checked [codenarc](https://codenarc.github.io/CodeNarc/codenarc-rules-formatting.html#spacearoundmapentrycolon-rule) so I could configure it but regex isn't allowed in a json file (also tried and see if escaping it works but it doesn't. So if my config is like the one below:
```
{
    "extends": "recommended",
    "rules": {
        "comments.ClassJavadoc": "off",
        "formatting.Indentation": {
            "spacesPerIndentLevel": 2,
            "severity": "info"
        },
        "formatting.SpaceAroundMapEntryColon": {
            "characterBeforeColonRegex": "\s"
        }
    }
}
```
I get:
`Parse options error: Cannot read config file: /home/pjhrobles/gaf/.groovylintrc.json
Error: Unexpected token s in JSON at position 296`

As mentioned above, I tried escaping but it doesn't work.

Is there anyway to override or do I have to use `--codenarcargs` or `--rulesets`?

**Comment by nvuillam** [comment]
To be honest, i always disable rule SpaceAroundMapEntryColon because it raises some errors looking as false positive :/ 
It will be probably solved with the next version of CodeNarc, but it's not released yet

I would try

```json
{
    "extends": "recommended",
    "rules": {
        "comments.ClassJavadoc": "off",
        "formatting.Indentation": {
            "spacesPerIndentLevel": 2,
            "severity": "info"
        },
        "formatting.SpaceAroundMapEntryColon": {
            "characterBeforeColonRegex": "\\s"
        }
    }
}
```

or even simpler: :) 

```json
{
    "extends": "recommended",
    "rules": {
        "comments.ClassJavadoc": "off",
        "formatting.Indentation": {
            "spacesPerIndentLevel": 2,
            "severity": "info"
        },
        "formatting.SpaceAroundMapEntryColon": "off"
    }
}
```

**Comment by nvuillam** [comment]
@PJHRobles does my workaround work ?

**Comment by PJHRobles** [comment]
I haven't gotten back to it and just disabled the rule for now. Will let you know when I do and if it works

**Comment by nvuillam** [comment]
Thanks for your feedback, i close the issue until the new version of CodeNarc :)
